src/commonComponents/plusAndMinusButton/PlusAndMinusButton.js

```diff
@@ -0,0 +1,48 @@
+//Libraries
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//Helpers
+import _noop from 'lodash/noop';
+
+//CSS
+import './plusAndMinusButton.css';
+
+function PlusAndMinusButton(props) {
+  const { label, onClick, className, id } = props;
+
+  function handleIncrement() {
+    return onClick(id, 1);
```

Comment:
It should invoke the callback fn just with 1, it should not keep track of id, parent component should take care of it